### DISCUSSION TITLE
KB5023706 Conflicts with Open-Shell?

### DISCUSSION THREAD

**Discussion by MACinBell** [Open-Shell/Open-Shell-Menu#1359] [Q&A]
I have 4.4.170 Open Shell and Explorer Patcher install.  After installing W11 KB5023706, Windows would go through the booting process, but the UI would not come up.  Instead, the dual screens I have just kept flashing between the desktop and a white screen.  I ran all the usual Command com diagnostics using boot as well as memory tests, but that made no difference.  The only thing that would semi-work in Windows was Task Manager.  After doing a restore to pre-KB5023706, all was well.   As soon as I reinstalled KB5023706, the problem reappeared.  This sure looks like a user interface/explorer issue.  Anyone else have a similar problem with open shell/explorer patcher?  Is the culprit more likely to be EP?

**Comment by MACinBell** [comment]
Update:  Looks like explorerpatcher was the problem.  But I also see that MS is issuing warnings about programs that alter the W11 UI, so I assume that would include Open Shell as well.

**Comment by tallman69** [comment]
ExplorerPatcher is the problem.  The second to the last update to ExplorerPatcher addresses the issue.  If you can boot into safe mode you may be able to uninstall ExplorerPatcher, then install the latest version after you are able to boot Windows normally. This issue was discussed some in this thread earlier:  https://github.com/Open-Shell/Open-Shell-Menu/discussions/1340

**Comment by tallman69** [comment]
If you don't already know, this explains how to uninstall or install a program in safe mode:  https://www.tenforums.com/tutorials/143263-how-install-uninstall-programs-safe-mode-windows-10-a.html

**Reply by MACinBell** [reply]
Yes, as I indicated, I did discover that ExplorerPatcher was the problem.  After I did that, it was relatively easy to uninstall EP.  I was also aware that you can uninstall a program using the command prompt in safe mode. The difficulty is it took me about 4 hours to figure out which program was causing the problem.  I initially suspected OpenShell because it does a lot more than EP, but I uninstalled EP first hoping it was the cause and that did the trick.  There is a pending additional release of EP to deal with potential future looping problems in case it occurs again with new W11 updates.  I think part of the issue is whether to take the warning from Microsoft that programs that create a classic UI are risky.  MS has recommended that they not be installed.

https://winaero.com/microsoft-officially-recommends-avoiding-apps-that-restore-classic-features-in-windows-11/
